endpoints/btc/index.js

```diff
@@ -0,0 +1,41 @@
+const request = require('request')
+const app = require('../../server')
+
+// set cache time to 1 minute
+const cacheTime = 60
```

Comment:
Can we move down the caching timer to make the API more useful? Will the source hit a rate limit if we move it to 10s for example?

Reply:
10 seconds is fine.